component/component.go

```diff
@@ -76,7 +76,7 @@ type Host interface {
 	// Typically is used to find exporters by type or by full config name. Both cases
 	// can be done by iterating the returned map. There are typically very few exporters
 	// so there there is no performance implications due to iteration.
-	// This returns a map by DataType of maps by exporter configs to the exporter instance.
```

Comment:
Is this change intended (+ a few other similar ones in this PR?)

Reply:
done. goland decided to do these changes because of a rename.